python/rpdk/java/templates/init/guided_aws/ClientBuilder.java

```diff
@@ -0,0 +1,8 @@
+package {{ package_name }};
+
+public class ClientBuilder {
+  // sample client builder - change ServiceSdkClient on the desired one
+  public static ServiceSdkClient getClient() {
```

Comment:
Should this be commented out ? Otherwise it will not build right ?

Reply:
it wont build. previous commit has a buildable version but it's less explicit